src/settings/SettingPage/getCRUDActions.js

```diff
@@ -0,0 +1,106 @@
+import { buildUrl } from '@folio/stripes/smart-components';
+
+export const getCRUDActions = ({
+  finishedResourceName,
+  mutator: { [finishedResourceName]: resourceMutator },
+  location,
+  history,
+  match: { path },
+  onUpdateRecordError,
+  onDeleteSuccess,
+  onDeleteError,
+}) => {
+  const transitionToParams = params => {
+    const url = buildUrl(location, params);
+
+    history.push(url);
+  };
+
+  const createRecord = record => resourceMutator.POST(record)
+    .catch(error => {
+      onUpdateRecordError(record, error);
+
+      throw error;
+    });
+
+  const editRecord = record => resourceMutator.PUT(record)
+    .catch(error => {
+      onUpdateRecordError(record, error);
+
+      throw error;
+    });
+
+  // const showDeleteRecordSuccessfulMessage = record => {
```

Comment:
Will commented code to be changed/removed?

Reply:
Removed